src/fides/lib/models/client.py

```diff
@@ -114,6 +122,7 @@ def create_access_code_jwe(self, encryption_key: str) -> str:
             JWE_PAYLOAD_SCOPES: self.scopes,
             JWE_ISSUED_AT: datetime.now().isoformat(),
             JWE_PAYLOAD_ROLES: self.roles,
+            JWE_PAYLOAD_SYSTEMS: self.systems,
```

Comment:
When creating a token on login, add the `systems` (system_ids) from the client if applicable.